README.md

```diff
@@ -86,6 +86,8 @@ environment.
 There are a couple of ways to inform balena CLI that it should use a different
 environment.
 
+*Please note that environments are isolated and thus you can't use the same user account interchangeably. You need to create a different account for each environment, thus before proceeding with the guide ensure that you have an account for balena-staging.*
```

Comment:
I'd suggest actually having a link for balena-staging here which goes to the https://dashboard.balena-staging.com site, to make it clear.